src/java.base/share/classes/java/security/Policy.java

```diff
@@ -896,5 +896,15 @@ public UnsupportedEmptyCollection() {
         @Override public Enumeration<Permission> elements() {
             return perms.elements();
         }
+
+        /**
+         * If this object is readonly, no new objects can be added to it using {@code add}.
+         *
+         * @return {@code true} if this object is marked as readonly, {@code false} otherwise.
+         */
+        @Override
+        public boolean isReadOnly() {
+            return perms.isReadOnly();
```

Comment:
You could just return true here. Either way is ok with me.

Reply:
Yeah, but since we do call `perms.setReadOnly()` in the constructor I figured I'll it this way.